front/components/assistant/conversation/UserMessage.tsx

```diff
@@ -1,19 +1,16 @@
 import { Avatar } from "@dust-tt/sparkle";
+import React from "react";
 
+import { ConversationMessage } from "@app/components/assistant/conversation/ConversationMessage";
 import { UserMessageType } from "@app/types/assistant/conversation";
 
 export function UserMessage({ message }: { message: UserMessageType }) {
   return (
-    <>
-      <div className="flex-shrink-0">
-        {message.user && <Avatar visual={message?.user.image} size="sm" />}
-      </div>
-      <div className="flex-grow">
-        <div className="flex flex-col gap-4">
-          <div className="text-sm font-medium">{message.context.fullName}</div>
-          <div className="text-base font-normal">{message.content}</div>
-        </div>
-      </div>
-    </>
+    <ConversationMessage
+      avatar={<Avatar visual={message.user?.image} size="sm" />}
```

Comment:
I find that API too complicated, as mentioned before. I don't know if it was part of the specs, but just raising my concern.